internal/indexmanagement/scripts.go

```diff
@@ -668,6 +670,11 @@ decoded=$(echo $PAYLOAD | base64 -d)
 # need to get a list of all mappings under ${POLICY_MAPPING}, drop suffix '-write' iterate over
 writeAliases="$(getWriteAliases "$POLICY_MAPPING")"
 
+if [ -z $writeAliases ]; then
+  echo "No indices to rollover"
```

Comment:
Slight change to keep in sync with other message (see other comment for reasoning):

```suggestion
  echo "No indices found. Skipping rollover."
```